README.md

```diff
@@ -329,9 +329,32 @@ current development is focused on djot.js, and it is possible
 that djot.lua will not be kept up to date with the latest syntax
 changes.
 
-Here are the relative render times for `pandoc-manual.dj`:
-
-![Render Times](/images/render-time.png)
+| Command | Mean [ms] | Min [ms] | Max [ms] | Relative |
+|:---|---:|---:|---:|---:|
+| `/home/runner/go/bin/godjot pandoc-manual.dj` | 1.1 ± 0.1 | 1.0 | 2.2 | 1.00 |
+| `djoths pandoc-manual.dj` | 34.1 ± 0.5 | 33.4 | 35.4 | 31.40 ± 1.91 |
+| `djot pandoc-manual.dj` | 127.3 ± 3.3 | 120.6 | 132.7 | 117.16 ± 7.60 |
+| `/home/runner/.luarocks/bin/djot pandoc-manual.dj` | 149.4 ± 2.0 | 146.8 | 152.4 | 137.58 ± 8.36 |
+| `jotdown pandoc-manual.dj` | 6.1 ± 0.1 | 5.8 | 6.7 | 5.58 ± 0.34 |
+
+Running on AMD EPYC 7763 64-Core Processor at 2024-12-03 01:23:59+00:00  
+go version go1.23.3 linux/amd64  
+Go djot godjot@v1.0.5  
+The Glorious Glasgow Haskell Compilation System, version 9.10.1  
+cabal-install version 3.12.1.0
+compiled using version 3.12.1.0 of the Cabal library  
+Haskell djot UNKNOWN  
+npm version 10.9.0  
+node version v22.11.0  
+JavaScript djot 0.3.1  
+Lua 5.4.4  Copyright (C) 1994-2022 Lua.org, PUC-Rio  
+/usr/local/bin/luarocks 3.11.1
+LuaRocks main command-line interface  
+Lua djot 0.2.1  
+cargo 1.83.0 (5ffbef321 2024-10-29)  
+hyperfine 1.19.0  
+Rust djot
+jotdown v0.6.0
```

Comment:
If this material is included, I'd recommend omitting the cabal-install, Cabal, npm, luarocks, cargo, hyperfine versions. They are irrelevant to the benchmarks, since they just concern installation.

Here is a way to get the djoths version: `grep "^version:" djot.cabal | awk '{print $2}'`.

Reply:
Also, if this material is included, it should be in a bulleted list or something, since in the current format it will be rendered as one big paragraph...

Reply:
There are spaces on the end of each line, so they get their own lines.

Reply:
I don't know where `djot.cabal` lives and when I run `cabal install djot`, `cabal info djot` says ` Versions installed: [ Not installed ]`.